vision/cloud-client/document_text/doctext.py

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env python
+
+# Copyright 2017 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Outlines document text given an image.
+
+Example:
+    python doctext.py resources/cropme.jpg
+"""
+# [START full_tutorial]
+# [START imports]
+import argparse
+from enum import Enum
+import io
+
+from google.cloud import vision
+from PIL import Image, ImageDraw
+# [END imports]
+
+
+class FeatureType(Enum):
+    PAGE = 1
+    BLOCK = 2
+    PARA = 3
+    WORD = 4
+    SYMBOL = 5
+
+
+def draw_boxes(im, blocks, color, width):
+    """Draw a border around the image using the hints in the vector list."""
+    # [START draw_blocks]
+    draw = ImageDraw.Draw(im)
+
+    for block in blocks:
+        draw.line([block.vertices[0].x, block.vertices[0].y,
+                  block.vertices[1].x, block.vertices[1].y],
+                  fill=color, width=width)
+        draw.line([block.vertices[1].x, block.vertices[1].y,
+                  block.vertices[2].x, block.vertices[2].y],
+                  fill=color, width=width)
+        draw.line([block.vertices[2].x, block.vertices[2].y,
+                  block.vertices[3].x, block.vertices[3].y],
+                  fill=color, width=width)
+        draw.line([block.vertices[3].x, block.vertices[3].y,
+                  block.vertices[0].x, block.vertices[0].y],
+                  fill=color, width=width)
+
+    return im
+    # [END draw_blocks]
+
+
+def get_document_bounds(image_file, feature):
+    # [START detect_bounds]
+    """Returns document bounds given an image."""
+    vision_client = vision.Client()
+
+    bounds = []
+
+    with io.open(image_file, 'rb') as image_file:
+        content = image_file.read()
+
+    image = vision_client.image(content=content)
+    document = image.detect_full_text()
+
+    for b, page in enumerate(document.pages):
```

Comment:
I didn't come up with anything clever for this, but can you at least leave a comment?

Also, no need to use `enumarate` here, just:

```python
for page in document.pages
```

ad nauseam. 

(also I feel like you did this in another sample, maybe revisit it?)